.ado/jobs/integration-test.yml

```diff
@@ -106,7 +106,7 @@ jobs:
       - script: yarn integration-test
         displayName: yarn integration-test
         workingDirectory: packages/integration-test-app
-        condition: eq(variables.NoDeployOption, '')
+        condition: and(succeeded(), eq(variables.NoDeployOption, ''))
```

Comment:
This will only ensure that the integration test only runs if the screenshot succeeds right? The actual build is too many steps behind this.